deployment/articles/03-installing-a-search-engine/02-elasticsearch/09-upgrading-to-elasticsearch_68.markdown

```diff
@@ -0,0 +1,58 @@
+---
+header-id: upgrading-to-elasticsearch-6-8
+---
+
```

Comment:
@lipusz Do you think we should have Upgrading to Elasticsearch 6.5 AND Upgrading to Elasticsearch 6.8 articles?

These articles already contain very little actual information. However, if you think it will help deflect some support requests, it's fine.

Reply:
Because of the embedded that remains 6.5.0 I'd keep both if you don't mind.